### ISSUE TITLE
About the calculation of virtual adversarial loss

### ISSUE THREAD

**Issue by lizezheng** [tensorflow/models#2484]
After reading the code,  wondering why we calculate the virtual adversarial loss with LM_EMBEDDED which used for pretraining  instead of CL_EMBDDED?

**Comment by lizezheng** [comment]
it seems that during the training time, calculate the virtual adversarial loss with CL_EMBEDDED is more reasonable ?

**Comment by asimshankar** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
